### PR TITLE
[044] connect form and internship by creating

### PR DESCRIPTION
**Pull request by Maksim-maevsky**
- added parameter internshipID for creating form

### CODE REVIEW

**Review by Fenydude — DISMISSED** [review]

File: src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java [hunk 1]
```diff
@@ -53,17 +54,23 @@ public FormFullDTO process(FormRegisterDTO form, MultipartFile file) {
...
+    private FormFullDTO saveForm(FormRegisterDTO formRegisterDTO, String internshipIdStr) {
+
+        UUID internshipId = UUID.fromString(internshipIdStr);
 
         Form form = mapper.toFormEntity(formRegisterDTO);
+
+        log.info("Save form, id: {} internshipId {} and status {}", form.getId(), internshipIdStr, FormStatus.REGISTERED);
+
         form.setFormStatus(FormStatus.REGISTERED);
+        form.setInternshipId(internshipId);
```
Comment (kzaretskaya) [inline src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java]: It's UI responsibility to set `internshipId`

File: src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java [hunk 2]
```diff
@@ -37,14 +37,15 @@
     @Value("${file.path}")
     private String filePath;
 
-    public FormFullDTO process(FormRegisterDTO form, MultipartFile file) {
+    public FormFullDTO process(FormRegisterDTO form, MultipartFile file, String internshipIdStr) {
```
Comment (SergeyKoval) [inline src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java]:
```suggestion
    public FormFullDTO process(FormRegisterDTO form, MultipartFile file, String internshipId) {
```

File: src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java (hunk 1 above)
Comment (Maksim-maevsky) [reply src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java]: done

File: src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java (hunk 2 above)
Comment (Maksim-maevsky) [reply src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java]: i removed this one

**Review by OlgaEvdokimova — APPROVED** [review]

**Review by SergeyKoval — APPROVED** [review]

**Pull request merged**